docs/security/tls_setup.md

```diff
@@ -0,0 +1,145 @@
+# TLS support for Percona Everest
+
+The Percona Everest server now supports Transport Layer Security (TLS)  for all incoming connections. TLS **encrypts** client and server communication. This keeps data safe from being **intercepted or changed**. Administrators can configure server certificates and private keys to enable **secure HTTPS access** to the Everest API server. This strengthens security for production environments.
+
+!!! note
+    - When TLS is enabled, the default server port (8080) will only accept `https` traffic, and `http` traffic will be rejected.
+    - We do not support redirects from `http` to `https`.
+    - Self-signed certificates aren't trusted by most browsers, so it's best to use a trusted certificate from a Certificate Authority (CA) in production.
+
+
+
+## TLS setup with Percona Everest
+
+!!! info "Important"
+    This section provides an example using **Helm**. You can also use the provided options with `everestctl` by using the flag `—helm.set`.
+
+
+### Use Cert-manager (recommended)     
+
+#### Prerequisites
+
+- Ensure that [cert-manager](https://cert-manager.io/docs/) has been deployed in your infrastructure.
+
+- Ensure that you have a properly configured **Issuer** or **ClusterIssuer** in place. See the [documentation](https://cert-manager.io/docs/configuration/) for details.
+
```

Comment:
when I open these links, it opens up in the same tab. Can we do it so that the links open in a new tab?

Reply:
Yes. Will add it.